painttiming.bs

```diff
@@ -62,9 +87,34 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn>Element or pseudo-element</dfn>: A [=document-sourced element=] or a [=generated content pseudo-element=].
+<dfn>Contentful</dfn>: An [=element or pseudo-element=] counts as contentful when one or more of the following apply:
+* The element has a [=text node=] child, with at least one character, excluding [=document white space characters=].
+* The element is a [=replaced image=] representing a [=loaded=] image.
+* The element's [=computed=] style contains a [=loaded=] [=background-image=].
+* The element is a [=canvas=] with its [=context mode=] set to any value other than <code>none</code>.
+* The element is an [=svg element with rendered descendants=].
+* The element's [=computed=] style contains a [=contentful=] and [=paintable=] [=generated content pseudo-element=].
+
+<dfn>Paintable</dfn>: As a general rule, an [=element or pseudo-element=] is paintable if it is within the viewport, or can potentially be in the viewport as a result of scrolling.
```

Comment:
scrolling or zooming I suppose? But I'm not certain that such a phrase in something that should occur in the normative text. I think this clarification is best left in a non-normative informal note instead.

Reply:
ok, will change to a note.